config/crd/bases/rabbitmq.com_rabbitmqclusters.yaml

```diff
@@ -4598,6 +4598,10 @@ spec:
                   description: Specify any rabbitmq advanced.config configurations
                   maxLength: 100000
                   type: string
+                envConfig:
+                  description: Modify to add to the rabbitmq-env.conf file. Modify this property on an existing RabbitmqCluster will trigger a StatefulSet rolling restart and will cause rabbitmq downtime.
```

Comment:
I believe the second "Modify" should actually be "Modifying"

Reply:
Done